command/lock.go

```diff
@@ -101,6 +104,9 @@ func (c *LockCommand) run(args []string, lu **LockUnlock) int {
 			"is generated based on the provided child command.")
 	f.BoolVar(&passStdin, "pass-stdin", false,
 		"Pass stdin to the child process.")
+	f.BoolVar(&shell, "shell", false,
```

Comment:
If we are going to move health checks away from using the shell (via having the user put the shell in themselves via `args` then we could omit this part of the change.